inspire_disambiguation/api.py

```diff
@@ -165,6 +201,19 @@ def cluster(ethnicity_model_path, distance_model_path, n_jobs, signature_block=N
             if hasattr(cluster, "supervised_scoring")
             else None,
         )
+    if test_signatures_uuids:
+        training_statistics = cluster.score(test_signatures_uuids, test_labels)
+        (
+            (B3_statistics_all, wrongly_classified_pairs),
+            B3_statistics_training,
+            B3_statistics_test,
+        ) = training_statistics
+        LOGGER.info(
+            B3_precision_recall_f_score_all=str(B3_statistics_all),
+            B3_precision_recall_f_score_training=str(B3_statistics_training),
+            B3_precision_recall_f_score_test=str(B3_statistics_test),
```

Comment:
it would be clearer for each of them to be a `dict` with `precision=...`, etc. instead of stringifying a tuple here.